examples/benchmark_problems/SAS_modelling/1D/prob_def_1.txt

```diff
@@ -4,4 +4,4 @@ name = 'Problem Def 1'
 description = 'An example data set for SasView 1D data'
 input_file = 'cyl_400_20.txt'
 function ='name=cylinder,radius=35.0,length=350.0,background=0.0,scale=1.0,sld=4.0,sld_solvent=1.0'
-parameter_ranges = {'radius': (1,50), 'length': (1,500)}
\ No newline at end of file
+parameter_ranges = {'radius': (30,40), 'length': (1,500)}
```

Comment:
Assuming this change has been accidentally saved

```suggestion
parameter_ranges = {'radius': (1,50), 'length': (1,500)}
```

Reply:
Good spot -- thanks!